test/utils.ts

```diff
@@ -17,3 +17,5 @@ export function mockCaptchaValidationOnce(action: RecaptchaActionType) {
     score: 1,
   });
 }
+
+export const tokenRegex = /\?t=([\w\-\.]{1,1000})/;
```

Comment:
This should be in ALL_CAPS.

Also where does the `{1,1000}`come from ? Is it arbitrary ?

Reply:
OK for all caps. The limit was indeed arbitrary, I've changed it to `+`.